run/idp-sql/pom.xml

```diff
@@ -40,14 +40,14 @@ limitations under the License.
         <!-- Import dependency management from Spring Boot -->
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-dependencies</artifactId>
-        <version>2.7.11</version>
+        <version>3.1.0</version>
```

Comment:
Here we're specifying `3.1.0` but `3.0.6` was used for other Spring Boot dependencies. Should we the same version across-the-board in this PR, and if so, should we upgrade to the latest while we're at it?